spring-cloud-starter-task-composedtaskrunner/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-cloud-starter-task-composedtaskrunner</artifactId>
+    <packaging>jar</packaging>
+    <name>Spring Cloud Starter Task Composed Task Runner</name>
+    <description>Contains the app for the Composed Task Starter</description>
+
+    <parent>
+        <groupId>org.springframework.cloud.task.app</groupId>
+        <artifactId>composedtaskrunner-task-app-starters-build</artifactId>
+        <version>1.1.1.BUILD-SNAPSHOT</version>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-task-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-task-batch</artifactId>
+        </dependency>
```

Comment:
Why are we bringing in the `spring-cloud-task-core` and `spring-cloud-task-batch` explicitly instead of using our starter?